tweepy/asynchronous/streaming.py

```diff
@@ -3,6 +3,7 @@
 # See LICENSE for details.
 
 import asyncio
+from asyncio import TimeoutError
```

Comment:
This import won't be required as you have referenced `TimeoutError` as `asyncio.TimeoutError`.